lib/crystalball/extensions/git/base.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Git
+  class Base # rubocop:disable Style/Documentation
+    def merge_base(from, to)
+      lib.merge_base(from, to)
+    end
```

Comment:
merge-base is a commit so we should return a commit object instead of just string. See `Git::Base::Factory#commit_tree` for example. Then we can do `repo.merge_base('HEAD^", "HEAD").sha` to get commit hash.

Reply:
I think gcommit will be better